### PR TITLE
Add test data and cucumber tests for negative test scenarios

### PR DESCRIPTION
**Pull request by sw1ld**
# Summary

 - PIS/AIS unsuccessful SCA
 - PIS service blocked

First part of negative cases, more to come.

# Checklist for Definition of Done

* [x] Code is technically reviewed by myself
* [x] Code is manually tested
* [x] Pipeline is green
* [x] Automatic tests for new functionality are created
* [x] ~~Documentation is updated~~
* [x] Branch is rebased and therefore ready to merge
* [x] Git history is clean

### CODE REVIEW

File: service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AIS.feature [hunk 1]
```diff
@@ -38,15 +29,16 @@ Feature: AIS
...
-    #TODO waiting for XS2A-Core patch
-#  Scenario Outline: Consent Status Valid
-#    Given PSU created a consent on dedicated accounts for account information <accounts>, balances <balances> and transactions <transactions>
-#    And PSU authorised the consent with psu-id <psu-id>, password <password>, sca-method <sca-method> and tan <tan>
-#    When PSU requests the consent status
-#    Then the appropriate status <status> and response code <code> are received
-#    Examples:
-#      | accounts               | balances               | transactions           | psu-id | password | sca-method | tan   | status | code |
-#      | DE94500105178833114935 | DE94500105178833114935 | DE94500105178833114935 | PSU-1  | 12345    | SMS_OTP    | 54321 | valid  | 200  |
+ # TODO Blocked by a bug in xs2a, waiting for hotfix for getAccounts endpoint
```
Comment (wandi34) [inline service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AIS.feature:32]: There should be no bug which blocks this test. Worked after update 1.15

File: service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AIS.feature [hunk 2]
```diff
@@ -16,15 +16,6 @@ Feature: AIS
       | null                                          | DE94500105178833114935 | null                   | 200  |
       | DE94500105178833114935;DE96500105179669622432 | DE94500105178833114935 | null                   | 200  |
 
-    #TODO waiting for XS2A-Core patch
```
Comment (wandi34) [inline service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AIS.feature:19]: Why deleting the whole test?
